src/generator/CodeGenerator.php

```diff
@@ -26,20 +26,26 @@ public function __construct($config = null) {
 			$this->config = $config;
 		} else {
 			$this->config = new CodeGeneratorConfig();
-		} 
+		}
 
 		$this->init();
 	}
-	
+
 	protected function init() {
 		if ($this->config->getGenerateEmptyDocblock()) {
```

Comment:
That might be a better approach indeed. If we would do that I would get rid of the EmptyDocblockVisitor class as well and also do that option wise in the DefaultVisitor class. However that does change the way the whole thing is set up now.